docker/planet/docker-entrypoint.sh

```diff
@@ -10,7 +10,13 @@ else
   PROTOCOL="http"
 fi
 
-sed -i -e "s#planet-db-host#$PROTOCOL://$DB_HOST#g" /usr/share/nginx/html/main*
+if [ "$ON_RPI" = "true" ]
```

Comment:
does docker-entrypoint runs on every request? As it seems to be using window.location which is supposed to be from browser